tests/installation/bootloader_zkvm.pm

```diff
@@ -71,8 +72,12 @@ sub run() {
     my $size_i = get_var('HDDSIZEGB') || '4';
 
     if (my $hdd = get_var('HDD_1')) {
+        my $hdd_dir  = "/var/lib/openqa/share/factory/hdd";
```

Comment:
I don't like this. This would not work with more webuis. I'm for something like this
```perl
my $hdd_dir = catdir(get_var('PRJDIR'), 'factory/hdd');
```

This should work even when s390 is adapted to cache the images because AFAIK PRJDIR is changed to correct dir for cache.